itest/rust/src/builtin_tests/containers/dictionary_test.rs

```diff
@@ -615,3 +615,19 @@ fn dictionary_iter_erase() {
     ```
      */
 }
+
+#[itest]
+fn dictionary_should_format_with_display() {
+    assert_eq!(format!("{}", Dictionary::new()), "{  }");
+    assert_eq!(
+        format!(
+            "{}",
+            dict! {
+                "one": 1,
+                "two": true,
+                "three": Variant::nil()
+            }
+        ),
+        "{ one: 1, two: true, three: <null> }"
+    )
+}
```

Comment:
Also this would benefit from extracting expressions into variables. Ideally the assert statements are single-line and not nested.

Reply:
Will Do.